frontend/components/ClubPage/EventCarousel.tsx

```diff
@@ -0,0 +1,149 @@
+import 'react-multi-carousel/lib/styles.css'
+
+import React, { useState } from 'react'
+import Carousel from 'react-multi-carousel'
+import styled from 'styled-components'
+
+import { ClubEvent } from '../../types'
+import { Icon, StrongText } from '../common'
+import Modal from '../common/Modal'
+import EventCard from '../EventPage/EventCard'
+import EventModal from '../EventPage/EventModal'
+
+const LeftArrow = styled.div`
+  left: -10px;
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+  z-index: 100;
+  opacity: 0.6;
+  transition-duration: 300ms;
+  cursor: pointer;
+  &:hover {
+    opacity: 1;
+  }
+`
+const RightArrow = styled.div`
+  right: -10px;
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+  z-index: 100;
+  opacity: 0.6;
+  transition-duration: 300ms;
+  cursor: pointer;
+  &:hover {
+    opacity: 1;
+  }
+`
+
+const CarouselWrapper = styled.div`
+  position: absolute;
+  box-sizing: border-box;
+  width: 100%;
+`
+const EventWrapper = styled.div`
+  @media only screen and (max-width: 580px) {
+    margin: 25px;
+  }
+  @media only screen and (min-width: 580px) and (max-width: 900px) {
+    margin: 8px;
+  }
+  @media only screen and (min-width: 900px) and (max-width: 1400px) {
+    margin: 12px;
+  }
+  @media only screen and (min-width: 1400px) {
+    margin: 5px;
+  }
+`
+
+type EventsProps = {
+  data: ClubEvent[]
+}
+
+const EventCarousel = ({ data }: EventsProps) => {
+  const [show, setShow] = useState(false)
+  const [modalData, setModalData] = useState<ClubEvent>()
+
+  const showModal = (entry: ClubEvent) => {
+    setModalData(entry)
+    setShow(true)
+  }
+  const hideModal = () => setShow(false)
+  const responsive = {
+    superLargeDesktop: {
+      breakpoint: { max: 4000, min: 1400 },
+      items: 3,
+    },
+    desktop: {
+      breakpoint: { max: 1401, min: 1400 },
```

Comment:
Is this breakpoint needed? Or should all breakpoints be modified to account for `desktop` width?